src/com_tjvendors/admin/models/fields/timecreated.php

```diff
@@ -15,6 +10,11 @@
 
 defined('JPATH_BASE') or die;
 
+use Joomla\CMS\Form\FormField;
+use Joomla\CMS\Factory;
+use Joomla\CMS\Date\Date;
+use Joomla\CMS\Language\Text;
+
 jimport('joomla.form.formfield');
```

Comment:
remove jimport('joomla.form.formfield'); from all field files if related class is loaded

Reply:
done